lib/reverse_sentence.rb

```diff
@@ -1,6 +1,41 @@
 # A method to reverse the words in a sentence, in place.
```

Comment:
Comprehension questions:  The basic sorts are O(n^2) although BubbleSort and InsertionSort have good best-case versions.  MergeSort is O(n log n)